service/controller/v24/resource/configmap/desired.go

```diff
@@ -96,8 +98,13 @@ func (r *Resource) newConfigMap(customResource v1alpha1.KVMConfig, template stri
 			ObjectMeta: apismetav1.ObjectMeta{
 				Name: key.ConfigMapName(customResource, node, prefix),
 				Labels: map[string]string{
-					"cluster":  key.ClusterID(customResource),
-					"customer": key.ClusterCustomer(customResource),
+					// TODO: Delete two legacy labels from next release
```

Comment:
Why is this TODO here when you are backporting?

Reply:
it's on v25, v26 already. Do you think it's enough to have TODO only on WIP?

Reply:
Yes lets remove it. We are only backporting because the bug affects production clusters. 

We won't fix v24 so better to just have the comment in WIP versions.